docs/concepts.md

```diff
@@ -364,56 +364,45 @@ ignore the details while rebuilding the core simulator framework. A lot of impro
-TrafficSource -+-> RequestsBuffered -+-> RequestsProcessing --> RequestsComplete
-                                     |
-                                     +-> RequestsFailed
+
+TrafficSource -+-> RequestsRouting  -+-> RequestsProcessing -+-> RequestsComplete
+                                                             |       
+                                                             +-> RequestsFailed
 ```
 
 The diagram shows five possible Movements:
 
-* `arrive_at_buffer`, from TrafficSource to RequestsBuffered. These are scheduled
+* `arrive_at_routing`, from TrafficSource to RequestsRouting. These are scheduled
   during the creation of the simulation
-* `send_to_replica`, from RequestsBuffered to RequestsProcessing
+* `send_to_replica`, from RequestsRouting to RequestsProcessing
 * `complete_request`, from RequestsProcessing to RequestsComplete
-* `buffer_backoff`, from RequestsBuffered back into itself to simulate Activator behaviour
-* `exhausted_attempts`, from RequestsBuffered to RequestsFailed, representing a timeout
+* `request_failed`, from RequestsProcessing to RequestsFailed
 
-The happy path (`arrive_at_buffer`, `send_to_replica`, `complete_request`) is linear.
+The happy path (`arrive_at_routing`, `send_to_replica`, `complete_request`) is linear.
 However, this is true only in aggregate: the exact path for each Request can vary. While
-all Requests will come from the TrafficSource and all of them will spend at least one
-Movement in RequestsBuffered, the precise RequestsProcessing / RequestsComplete stocks
+all Requests will come from the TrafficSource and all of them will spend one
+Movement in RequestsRouting, the precise RequestsProcessing / RequestsComplete stocks
 are not known when the original Request arrival is scheduled. There are multiple
 RequestsProcessing stocks, each belonging to a Replica.
 
 There are two alternative paths.
 
-Initially, if a Request arrives but cannot find an active Replica, the RequestsBuffer
-emulates Activator logic by scheduling Movements back into itself with exponential
-backoffs. On each such Movement it checks to see if any Replicas are in the
-ReplicasActive stock. If it finds any, it will pick one with a round-robin scheme and
+Initially, if a Request arrives, it's sent to RequestRouting. From RequestRouting we schedule a Movement to 
+ReplicasActive stock. On each such Movement it picks a replica with a round-robin scheme and
 schedule a Movement of the Request to its RequestsProcessing. The RequestsProcessing
-stock will itself schedule a Movement into RequestsComplete.
-
-If the Request does not find any active Replicas, it is repeatedly rescheduled for the
-`buffer_backoff` Movement. After 18 such Movements -- 18 attempts is the hardcoded figure
-in the Activator -- the RequestsBuffer stock will instead schedule a Movement into
-RequestsFailed, representing timeouts.
+stock will itself schedule a Movement into RequestsComplete or RequestsFailed, representing timeouts.
 
```

Comment:
I think it's out of the scope this documentation. Because it's written in a high level and it didn't reflect the internal logic of request processing based on replica max requests per second capacity. We could add a link to cpu model doc. What do you think?

Reply:
Ok. If this doc is too high-level to go into cpu model, sounds good to skip here.